gitmanager.py

```diff
@@ -315,7 +316,15 @@ def merge_pull_request(cls, pr_id, comment=""):
             raise ValueError("PR description is malformed. Blame a developer.")
         if pr_info["state"] != "open":
             raise ValueError("PR #{} is not currently open, so I won't merge it.".format(pr_id))
+        string = pr_info["title"]
         ref = pr_info['head']['ref']
+        string = regex.match(r".*?: \S+ (.*?)(?:\Z|\s*\(\?#)", str).group(1)
+        string = string.replace("\\", "")
+        is_watchlist = "watch" in pr_info["title"]
+        is_phone = string.isdigit()
+        reasons = chatcommands.check_blacklist(string, False, is_watchlist, is_phone)
```

Comment:
The goal of this PR is to prevent the tests that are in `blacklist_integrity_check()` in helpers.py from failing once the PR is merged. To accomplish that, tests similar to what are done in `add_to_blacklist()` in gitmanager.py need to be re-run on the added watch/blacklist entry. That is not what `chatcommands.check_blacklist()` does. The checks which `chatcommands.check_blacklist()` performs are not something that needs to be re-run on the entry that is in the PR. They have already been done when the PR was created. The user has already done a `-force`, or not, as is required to pass those checks.

So, there's no need to resolve your circular import, because the thing you're importing that's causing you a problem shouldn't be used or imported.